src/main/java/org/usfirst/frc1745/deepspace2019/subsystems/vision/Limelight.java

```diff
@@ -0,0 +1,121 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc1745.deepspace2019.subsystems.vision;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableEntry;
+
+/**
+ * Add your docs here.
+ */
+public class Limelight extends Subsystem {
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+  private static NetworkTableInstance networkTableInstance;
+
+  public Limelight(){
+    networkTableInstance = NetworkTableInstance.getDefault();
+    networkTableInstance.startClientTeam(1745);
+    networkTableInstance.startDSClient();
+  }
+
+  @Override
+  public void initDefaultCommand() {
+
+    // do nothing
+  }
+
+  // Constants to be adjusted for calculations
+  private double KpAim = -0.02;
+  private double KpDistance = -0.02;
+  private double min_aim_command = 0;
```

Comment:
Likewise, can we make `min_aim_command` camelCase instead of snake_case
so `min_aim_command` -> `minAimCommand`